src/coq_elpi_HOAS.ml

```diff
@@ -908,9 +910,10 @@ let purge_algebraic_univs_sort state s =
 
 let in_elpi_flex_sort t = E.mkApp sortc (E.mkApp typc t []) []
 
-let sort = { sort with API.Conversion.embed = (fun ~depth state s ->
+(* WIP: I do not know how to make this optional *)
```

Comment:
I guess the only way is to make this a ContextualConversion, in that case you receive a `{ option }` rather than just a state. I may try to fix that tomorrow if time allows for it.